package.json

```diff
@@ -5,15 +5,15 @@
   "main": "index.js",
   "dependencies": {
     "csslint": "^0.10.0",
-    "event-stream": "^3.1.2",
-    "gulp-util": "^3.0.0"
+    "event-stream": "^3.3.0",
+    "gulp-util": "^3.0.4"
   },
   "devDependencies": {
-    "mocha": "^1.18.2",
-    "should": "^3.3.1"
+    "mocha": "^2.2.1",
+    "should": "^5.2.0"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha"
+    "test": "mocha"
```

Comment:
``npm run-script`` prefixes ``node_modules/.bin`` to ``$PATH``, so you're safe with this :smile: